pkg/machines/components/storagePools/storagePoolDelete.jsx

```diff
@@ -86,28 +127,48 @@ export class StoragePoolDelete extends React.Component {
     }
 
     render() {
-        const { storagePool } = this.props;
+        const { storagePool, vms } = this.props;
         const id = storagePoolId(storagePool.name, storagePool.connectionName);
         const volumes = storagePool.volumes || [];
+        const volumesUsed = areVolumesUsedIndependently(storagePool, vms);
+        const poolUsed = isPoolUsed(storagePool, vms);
 
         let defaultBody = (
             <div className='ct-form'>
                 { storagePool.active && volumes.length > 0 && <React.Fragment>
                     <label className='control-label'>
                         {_("Delete Content")}
                     </label>
-                    <label className='checkbox-inline'>
-                        <input id='storage-pool-delete-volumes'
-                            type='checkbox'
-                            checked={this.state.deleteVolumes}
-                            onChange={e => this.onValueChanged('deleteVolumes', e.target.checked)} />
-                        {_("Delete the Volumes inside this Pool")}
-                    </label>
+                    <div role="group">
+                        <label className='checkbox-inline'>
+                            <input id='storage-pool-delete-volumes'
+                                type='checkbox'
+                                checked={this.state.deleteVolumes}
+                                disabled={volumesUsed}
+                                onChange={e => this.onValueChanged('deleteVolumes', e.target.checked)} />
+                            {_("Delete the Volumes inside this Pool")}
+                        </label>
+                        {volumesUsed && (
+                            <OverlayTrigger overlay={ <Tooltip id="tip-inforec">{_("Volumes in this pool are used by virtual machine.")}</Tooltip> } placement="top">
+                                <span className="fa fa-lg fa-info-circle" />
+                            </OverlayTrigger>)}
+                    </div>
                 </React.Fragment>}
                 { !storagePool.active && _("Deleting an inactive Storage Pool will only undefine the Pool. Its content will not be deleted.")}
             </div>
         );
 
+        const showFooterWarning = () => {
+            if (poolUsed) {
+                return (
+                    <span id={`delete-${id}-idle-message`} className='idle-message'>
+                        <i className='pficon pficon-info' />
+                        <span>{_("Pool is used as disk's source attribute")}</span>
```

Comment:
This needs different wording, it's too technical.
In addition I think we shouldn't even open the dialog when the pools volumes are used. Just disable the button with a tooltip explaining the reason, exactly how we do that for storage volumes.

Reply:
For both cases, what about just use tooltip which says: "Pool's volumes are used by: ...."

Reply:
Sounds good to me.